tests/TestCase/Command/PluginAssetsCommandsTest.php

```diff
@@ -91,6 +91,29 @@ public function testSymlink(): void
         $this->assertTrue(is_link($path));
     }
 
+    /**
+     * testRelativeSymlink method
+     */
+    public function testRelativeSymlink(): void
+    {
+        $this->loadPlugins(['TestPlugin' => ['routes' => false], 'Company/TestPluginThree']);
```

Comment:
We may need to skip this test on windows because of reduced symlink support.

```suggestion
        $this->skipIf(DS === '\\', 'Cant perform operations with symlinks windows.');
        $this->loadPlugins(['TestPlugin' => ['routes' => false], 'Company/TestPluginThree']);
```